### DISCUSSION TITLE
Ability to modify class names

### DISCUSSION THREAD

**Discussion by amakhmudov** [tailwindlabs/tailwindcss#2669] [Help] [answered]
Is there any ability to change the default class names:
"**text-center**" into "**ta-c**" (which means short of "text-align: center").
I think, that would be more logical in assigning different css properties without knowing documentation and default class names of tailwindcss...

**Comment by simonswiss** [comment, marked as answer]
Hey!

[This comment](https://github.com/tailwindlabs/tailwindcss/issues/102#issuecomment-395751449) by Adam Wathan from a while back might be useful to you 👍
